tools/file-server/main.go

```diff
@@ -9,7 +9,7 @@ func main() {
 	fs := http.FileServer(http.FS(os.DirFS(os.Args[2])))
 	http.Handle("/", fs)
 	//nolint:gosec // G114: file server used for testing purposes only.
-	err := http.ListenAndServe(":"+os.Args[1], nil)
+	err := http.ListenAndServe("localhost:"+os.Args[1], nil)
```

Comment:
This just removes the popup notification asking if you want to allow network connections (now it just runs)

Reply:
TIL! Cool!